### DISCUSSION TITLE
cartridge browser

### DISCUSSION THREAD

**Discussion by Starbrush** [nesbox/TIC-80#2495] [Ideas]
I think a cartridge browser would be interesting in many situations, something similar to _splore_ from pico-8, easier to download games, discover, play, and run on other systems that would be impossible to run with the emulator

**Comment by Skeptim** [comment]
I don't know pico-8, isn't _splore_ similar to the `surf` command ?

**Reply by Starbrush** [reply]
I saw that it's similar, but it's a bit meh, the interface isn't very good, it lacks a it lacks a lot of features, but from what I saw it is being improved little by little,  but sorry, I didn't know this surf feature existed, It's not apparent on the website. I'll be close this idea
